src/services/registration.js

```diff
@@ -1,40 +1,65 @@
-const dbService = require('./regCheckDB');
-const users = require('../dummy/userDB');
+const mysql = require('mysql');
+require('dotenv').config();
 
-const createUser = item => new Promise((resolve, reject) => {
-  if (!dbService.checkUserName(item.username) || !dbService.checkPassword(item.password)) {
-    reject(new Error(400));
-  } else if (item.password !== item.confirmPsw) {
-    reject(new Error(400));
-  } else if (!dbService.containsUser(item)) {
-    const user = {
-      userId: 1,
-      username: item.username,
-      password: item.password,
-    };
-    users.userDB.push(user);
-    resolve(user.userId);
-  } else {
-    reject(new Error(500));
+const conn = mysql.createConnection({
+  host: process.env.DB_HOST,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASS,
+  database: process.env.DB_NAME,
+  port: process.env.DB_PORT,
+  multipleStatements: true,
+});
+
+const checkUserName = (input) => {
+  const user = /^[A-Za-z0-9]\w{0,}$/;
+
+  if (input && user.test(input)) {
+    return true;
   }
+  return false;
+};
+
+const checkPassword = (input) => {
```

Comment:
Same as above, not sure what the function is checking the input for

Reply:
same